### DISCUSSION TITLE
New FBA API Inbound v240320 - Bug with auto generated CSV file("print pack list.csv") when creating shipments with Path #1 (misaligned columns)

### DISCUSSION THREAD

**Discussion by MetaMan2023** [amzn/selling-partner-api-models#4399] [Q&A]
When creating shipments using Path #1 (left path) of the new API, the auto-generated "Print pack list.csv" file has a column alignment issue when downloading through Seller Central.

Specific issue:

There are two parts to the spreadsheet:  (A) the SKUs and quantity rows and (B) the Box identifiers / parameters section (weight, dims, box name, etc).

The two sections are misaligned by 1 column

For example in the SKU/QTY rows (A above) the "Box 1 units" column starts in row J of the spreadsheet... but in section (B) (box identifiers), the first box starts in column K.

The box identifier information needs to slide over one column.

This is important as it is the only currently usable information for us to attempt to match up boxes.

![spreadsheet download button](https://github.com/user-attachments/assets/cc682c2e-c873-48d5-bcec-2e43080550cc)
![spreadsheet box info](https://github.com/user-attachments/assets/932b3874-b253-4f81-86ee-3910a45702ba)

**Comment by puppsupr** [comment]
Hi,
In order to correctly resolve this issue, Please [open a support case](https://developer.amazonservices.com/support) following the [guidance](https://developer.amazonservices.com/support-guidelines) here so we can pursue the investigation as additional details are required from your end.

**Comment by MetaMan2023** [comment]
@puppsupr 

Thank you -- we have case #16799486781 open regarding the CSV misalignment issue (2024-12-09) if you require it to follow up.

**Comment by puppsupr** [comment]
Hi @MetaMan2023,
Have the issue been resolved ?
